WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -633,6 +703,18 @@ public void onPostChanged(OnPostChanged event) {
         }
     }
 
+    private final Runnable mSearchRunnable = new Runnable() {
+        @Override
+        public void run() {
+            SearchPostsPayload payload = new SearchPostsPayload(mSite, mSearchTerm, mSearchOffset);
```

Comment:
I know we have some checks beforehand, but do you think it makes sense to check if the `mSearchTerm` is empty here, especially to avoid some race conditions?